examples/v1beta1/README.md

```diff
@@ -1,391 +1,141 @@
-```
-
-- FPGA XGBoost Parameter Tuning example, [source](https://github.com/inaccel/jupyter/blob/master/lab/dot/XGBoost/parameter-tuning.py).
-
-```
-docker.io/inaccel/jupyter:lab
-```
-
-- MPI operator horovod mnist example, [source](https://github.com/kubeflow/mpi-operator/tree/master/examples/horovod).
-
-```
-docker.io/kubeflow/mpi-horovod-mnist
-```
-
-- XGBoost operator LightGBM dist example, [source](https://github.com/kubeflow/xgboost-operator/tree/master/config/samples/lightgbm-dist).
-
-```
-docker.io/kubeflowkatib/xgboost-lightgbm
-```
+## AutoML Algorithms
+
+The following examples show various AutoML algorithms in Katib.
+
+### Hyperparameter Tuning
+
+Check the [Hyperparameter Tuning](https://www.kubeflow.org/docs/components/katib/overview/#hyperparameters-and-hyperparameter-tuning)
+Experiments for the following algorithms:
+
+- [Random Search](./hp-tuning/random.yaml)
+
+- [Grid Search](./hp-tuning/grid.yaml)
+
+- [Bayesian Optimization](./hp-tuning/bayesian-optimization.yaml)
+
+- [Tree of Parzen Estimators (TPE)](./hp-tuning/tpe.yaml)
+
+- [Multivariate TPE](./hp-tuning/multivariate-tpe.yaml)
+
+- [Covariance Matrix Adaptation Evaluation Strategy (CMA-ES)](./hp-tuning/cma-es.yaml)
+
+- [Sobol's Quasirandom Sequence](./hp-tuning/sobol.yaml)
+
+- [HyperBand](./hp-tuning/hyperband.yaml)
+
+### Neural Architecture Search
+
+Check the [Neural Architecture Search](https://www.kubeflow.org/docs/components/katib/overview/#neural-architecture-search)
+Experiments for the following algorithms:
+
+- [Efficient Neural Architecture Search (ENAS)](./nas/enas-gpu.yaml)
+
+- [Differentiable Architecture Search (DARTS)](./nas/darts-gpu.yaml)
+
+### Early Stopping
+
+Improve your Hyperparameter Tuning Experiments with the following
+[Early Stopping](https://www.kubeflow.org/docs/components/katib/early-stopping/) algorithms:
+
+- [Median Stopping Rule](./early-stopping/median-stop.yaml)
+
+## Katib Python SDK Examples
+
+To learn more about Katib Python SDK check [this directory](./sdk).
+
+## Resume Katib Experiments
+
+You can use different resume policies in Katib Experiments. Follow
+[this guide](https://www.kubeflow.org/docs/components/katib/resume-experiment/)
+to know more about it. Check the following examples:
+
+- [Resume From Volume](./resume-experiment/from-volume-resume.yaml)
+
+- [Never Resume](./resume-experiment/never-resume.yaml)
+
+## Metrics Collector
+
+Katib supports the various metrics collectors and metrics strategies.
+Check the [official guide](https://www.kubeflow.org/docs/components/katib/experiment/#configuration-spec)
+to know more about it. In this directory you can find the following examples:
+
+- [File Metrics Collector](./metrics-collector/file-metrics-collector.yaml)
+
+- [Custom Metrics Collector](./metrics-collector/custom-metrics-collector.yaml)
+
+- [Metrics Collection Strategy](./metrics-collector/metrics-collection-strategy.yaml)
+
+## TODO (andreyvelich) Discuss about this name. Trial Settings
```

Comment:
In that case, does [Trial Training Containers](https://github.com/andreyvelich/katib/tree/refactor-example-folder/examples/v1beta1/trial-training-containers) folder where we add training code make sense ?
WDYT @johnugeorge about the name [`trial-template`] instead of [`trial-settings`] ?

Reply:
that sounds better